tabpfn_client/prompt_agent.py

```diff
@@ -46,7 +46,9 @@ def prompt_and_set_token(cls, user_auth_handler: "UserAuthenticationClient"):
             password = getpass.getpass(cls.indent(password_req_prompt))
             password_confirm = getpass.getpass(cls.indent("Please confirm your password: "))
 
-            user_auth_handler.set_token_by_registration(email, password, password_confirm)
+            validation_link = input(cls.indent("Please enter your validation link: "))
```

Comment:
isn't it actually a `secret`?

Reply:
Shall we keep it as a secret_key / secret_code or something!

Reply:
Updated to secret code for prompt